card/lib/card/cache.rb

```diff
@@ -16,7 +16,7 @@ def real_file_path(name)
   class Cache
 
     @@prepopulating     = [ 'test','cucumber' ].include? Rails.env
-    @@using_rails_cache = Rails.env =~ /^cucumber|test$/
+    @@using_rails_cache = Rails.env =~ /^cucumber|test$/ # why two versions of same test?
```

Comment:
I think the separate class vars may be defensible (because we have played with different configurations), but the two tests are pretty ugly ;)